src/main/java/Duke.java

```diff
@@ -1,10 +1,245 @@
+import java.time.DateTimeException;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeParseException;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.time.LocalTime;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.File;
```

Comment:
I think these import statements should be in alphabetical order! Like `.io` then `.time` then `util` (and the packages within each of those too). Checkstyle will be helpful in flagging it out :)